src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4938,15 +4941,20 @@ static void window_ride_colour_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi
 #pragma region Music
 
 static constexpr const uint8_t MusicStyleOrder[] = {
```

Comment:
When the new save format is here, the constants no longer mean anything. I think we should get rid of this hardcoded order and just sort the dropdown alphabetically.